billy-portugal/src/main/java/com/premiumminds/billy/portugal/services/builders/impl/PTSimpleInvoiceBuilderImpl.java

```diff
@@ -74,11 +72,11 @@ protected void validateInstance() throws BillyValidationException {
                 PTGenericInvoiceBuilderImpl.LOCALIZER.getString("field.clientType"));
         super.validateInstance();
 
-        if (i.getClientType() == CLIENTTYPE.CUSTOMER && i.getAmountWithTax().compareTo(new BigDecimal(1000)) >= 0) {
-            throw new BillySimpleInvoiceException("Amount > 1000 for customer simple invoice. Issue invoice");
+        if (i.getClientType() == CLIENTTYPE.CUSTOMER && i.getAmountWithTax().compareTo(new BigDecimal("3000")) >= 0) {
+            throw new InvalidAmountForDocumentTypeException("Amount > 3000 for customer simple invoice. Issue invoice");
```

Comment:
Isn't the value 1000€ for end customers for simplified invoices in Portugal?

Reply:
Indeed it is. I misread my notes and mixed up the values for PT and ES.